packages/backend/src/execution/executors/PgExecutor/PgExecutor.ts

```diff
@@ -9,8 +9,6 @@ import { hydrate } from './hydrate';
 import { SynthqlError } from '../../../SynthqlError';
 
 // Use the OIDs imported from pg.types to set custom type parsers
-types.setTypeParser(types.builtins.TIME, (value) => value);
```

Comment:
what happened here?

Reply:
Not much... I learned that TIME & TIMEZ are already being returned as string, hence no real need for specifying the custom parser